node_modules/oae-core/userdetails/css/userdetails.css

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+@media (min-width: 769px) and (max-width: 920px) {
+    #userdetails-modal .modal-dialog {
+        margin: 0 10px;
+        width: auto;
+    }
+}
+
+@media (min-width: 920px) {
```

Comment:
Can we use `modal-lg` instead of this?
http://getbootstrap.com/javascript/#modals